llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -8882,8 +8865,9 @@ VPRecipeBuilder::handleReplication(Instruction *I, ArrayRef<VPValue *> Operands,
   assert((Range.Start.isScalar() || !IsUniform || !IsPredicated ||
           (Range.Start.isScalable() && isa<IntrinsicInst>(I))) &&
          "Should not predicate a uniform recipe");
-  auto *Recipe = new VPReplicateRecipe(
-      I, make_range(Operands.begin(), Operands.end()), IsUniform, BlockInMask);
+  auto *Recipe =
+      new VPReplicateRecipe(I, make_range(Operands.begin(), Operands.end()),
+                            IsUniform, BlockInMask, getMetadataToPropagate(I));
```

Comment:
s/make_range(Operands.begin(), Operands.end())/Operands while we're here?

Reply:
This will need some further changes, done separately in https://github.com/llvm/llvm-project/pull/137798 (and the ArrayRef change)